astropy/coordinates/representation.py

```diff
@@ -744,15 +881,42 @@ def lat(self):
         """
         return self._lat
 
+    def unit_vectors(self):
```

Comment:
I think I'll let this stand -- in the context of representations, unit vectors is really the logical term...